app/assets/sass/unbranded.scss

```diff
@@ -1,30 +1,12 @@
-// Path to assets for use with the file-url function
-// in the govuk frontend toolkit's url-helpers partial
-$path: "/public/images/";
+@import "node_modules/govuk-frontend/settings/all";
 
-// Import GOV.UK elements from /govuk-modules/, this will import the frontend toolkit and some base styles.
-// Take a look in /govuk-modules/_govuk-elements.scss to see which files are imported.
-@import 'govuk-elements';
+// Override the govuk-frontend font stack
+$govuk-font-family: "HelveticaNeue", "Helvetica Neue", "Arial", "Helvetica", sans-serif;
 
-// If you need to create a page as part of your journey, but without GOV.UK branding
-// See localhost:3000/examples/unbranded/
-
-// Override the govuk_frontend_toolkit's font stack
-// https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/_font_stack.scss#L13
-$toolkit-font-stack: $helvetica-regular;
-
-.unbranded {
-  background: $white;
+// Remove canvas background colour, as is used with the GOV.UK Footer.
+$govuk-canvas-background-colour: $govuk-body-background-colour;
```

Comment:
no need for this as we're not including the footer

Reply:
If we don't do this we get a grey box at the bottom of the screen, since the `<body>` does not extend the same length as `<html>`